$MirOS: ports/www/lynx/patches/patch-src_LYLocal_c,v 1.1 2010/12/23 20:34:01 tg Exp $

	Fix spelling; suggested for upstream inclusion.

--- src/LYLocal.c.orig	Mon Feb 13 02:45:53 2012
+++ src/LYLocal.c	Sat Mar  3 18:20:22 2012
@@ -13,7 +13,7 @@
  *	Changed to use the library's HTList_* functions and macros for
  *	  managing the list of tagged file URLs.
  *	Keep track of proper level of URL escaping, so that unusual filenames
- *	  which contain #% etc. are handled properly (some HTUnEscapeSome()'s
+ *	  which contain #% etc. are handled properly (some HTUnEscapeSome()s
  *	  left in to be conservative, and to document where superfluous
  *	  unescaping took place before).
  *	Dynamic memory instead of fixed length buffers in a few cases.
@@ -1906,7 +1906,7 @@ static char *build_command(char *line,
 }
 
 /*
- * Perform file management operations for LYNXDIRED URL's.  Attempt to be
+ * Perform file management operations for LYNXDIRED URLs.  Attempt to be
  * consistent.  These are (pseudo) URLs - i.e., they should be in URL syntax: 
  * some bytes will be URL-escaped with '%'.  This is necessary because these
  * (pseudo) URLs will go through some of the same kinds of interpretations and
